src/views/settings.html

```diff
@@ -141,29 +153,94 @@ <h2 class="mb-3">Search Results Options</h2>
               <input
                 type="checkbox"
                 class="form-check-input"
-                name="enableOptionsMenuItem" 
-                id="settings_enableOptionsMenuItem">
+                name="enableOptionsMenuItem"
+                id="settings_enableOptionsMenuItem"
+              />
               <label
                 class="form-check-label"
-                for="settings_enableOptionsMenuItem">
+                for="settings_enableOptionsMenuItem"
+              >
                 Show link to extension options at the end of context menu
               </label>
             </div>
           </li>
         </ul>
       </div>
-
     </div>
   </form>
 
-  <form name="import_settings">
-    <h2 class="text-center mt-3">Export/Import Search Options</h2>
+  <h2 class="text-center mt-3">Ixport/Emport/Edit Search Options</h2>
 
-    <p>Use the text area bellow to transfer your search engine options from one install of Chrome to another effortlessly. Just copy the string from here to the options page in other install of Chrome (or vice-versa) and press 'Import'.</p>
+  <p>
+    Use "Import" button to import the settings from your system, "Export" button
+    to export the settings to your system and, "Edit" button to edit the
+    settings manually.
+  </p>
 
-    <textarea id="settings_json" class="form-control text-monospace text-small" rows="12"></textarea>
-    <div class="text-right">
-      <button class="btn btn-success" type="button" id="settings_saveImport">Import</button>        
+  <div class="d-flex justify-content-end">
+    <input class="d-none" id="settings_fileInput" , type="file" />
+    <button type="button" class="btn btn-success mr-2" id="settings_import">
+      <i class="fas fa-file-import mr-1"> </i> Import
+    </button>
+    <button class="btn btn-success mr-2" type="button" id="settings_export">
+      <i class="fas fa-file-export mr-1"></i> Export
+    </button>
+    <button class="btn btn-success mr-2" type="button" id="settings_edit">
+      <i class="fas fa-edit mr-1"></i> Edit
```

Comment:
Personally, I prefer `Edit manually` instead of `Edit`, but you can also ping Austin to check for SOC-friendly labels

Reply:
`Edit manually` sounds good to me as well :)